### DISCUSSION TITLE
Quarterly planner

### DISCUSSION THREAD

**Discussion by vasco9999** [kudrykv/latex-yearly-planner#65] [Ideas]
Hello, I like your work but I'm missing a quarterly planner where I could see the 3 months spread in the page and could write in the different days.

**Comment by kudrykv** [comment]
Any specific planner that is missing this?

**Comment by vasco9999** [comment]
Something like this

![Screenshot_20221206-162452](https://user-images.githubusercontent.com/119439720/205967176-c8d99b2e-5556-4484-8646-1fd5257e1307.png)

**Reply by kudrykv** [reply]
It has these quarterly pages

![CleanShot 2022-12-06 at 20 02 21](https://user-images.githubusercontent.com/12697803/205987519-c2af7285-b904-431d-9dd8-3f90923cfb18.png)

**Reply by kudrykv** [reply]
Oh, if you need _that specific layout_, then yeah, this planner doesn't have it. My vision of building what is currently there is that one doesn't always have a specific date for something but rather some reference. To be honest, I don't feel the template you posted.

Nevertheless, we can still implement it. But, unfortunately, it won't come soon, as I'm currently busy.
Another option is to change the template and recompile it yourself.
